webcompat/templates/contributors/report-bug.html

```diff
@@ -26,14 +26,12 @@ <h2 id="how" class="headline-1 highlight">How To Report An Issue?</h2>
           </p>
 
           <p>
-            @@insert here form screenshot@@
-            <img src="/img/contributors/report-bug_1-form-empty.png" style="max-height: 75vh; max-width: 100%;" alt="">
+            <img src="/img/contributors/report-bug_1-form-empty.png" style="max-height: 75vh; max-width: 100%;" alt="Image of an empty bug report form" />
```

Comment:
Let's land these, but maybe file a follow up issue for the inline styles being moved into a class?

Reply:
Filed #2329. I'm OK with this being follow up work, thoughts @zoepage?